pre_commit_hooks/maven-checkstyle.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if ! command -v mvn &>/dev/null; then
+  >&2 echo "Error: 'mvn' command not found"
+  exit 1
+fi
+
+mvn checkstyle:check
```

Comment:
`mvn` or `./mvnw`? Would be better to prefer `./mvnw` if it's there. Looks like this is what it does:

https://github.com/ejba/pre-commit-maven/blob/ca87dfea7c550d8ac7fc0c2a78589929154c7d89/pre_commit_maven/utils/maven.py#L28-L35